src/session/mod.rs

```diff
@@ -204,6 +204,24 @@ impl Session<Measuring> {
 
         session.verify(&digest, build, msr)
     }
+
+    /// Verifies the session's measurement against the AMD SP's measurement
+    /// using an externally generated digest.
+    pub fn verify_with_digest(
+        mut self,
```

Comment:
Clippy indicates that this variable doesn't have to be a mutable reference. Can you remove the `mut`?

Reply:
Fixed, thanks!